tests/integration/features/steps/integration_steps.py

```diff
@@ -598,7 +599,8 @@ def i_am_using_storage_provider_with_grpc_server(context, storage_provider, clie
     config["monitoring"] = {"monitoring_provider": "local"}
 
     config["checks"] = {
-        "health_check": "cql"
+        "health_check": "cql",
+        "enable_md5_checks": "false"
```

Comment:
In order to avoid duplications (which indeed make Sonar unhappy), we could define a global variable at the top:
```
config_checks = {"health_check": "cql", "enable_md5_checks": "false"}
```

And then use: 

```
     config["checks"] = config_checks
```

On these lines and also on line 731.

Reply:
Thanks that worked :)